modules/appeals_api/spec/support/rswag_config.rb

```diff
@@ -82,6 +55,33 @@ def config
 
   private
 
+  def tags
+    [].tap do |a|
+      a << { name: 'Higher-Level Reviews', description: '' }
+      a << { name: 'Notice of Disagreements', description: '' }
+      a << { name: 'Supplemental Claims', description: '' } if wip_doc_enabled?(:supplemental_claims)
```

Comment:
Also for the sake of demonstration?

Reply:
So the conditional should be the default in the future - as soon as we add a new tag/appeal/appeal version etc. we always put `if wip_doc_enabled?...` with the default to  
_false_ and then turn the switch on when ready? 

Am I thinking about this correctly?

Reply:
Pretty much correct, except it's driven by the mere presence of the flag to indicate what's WIP. There is no truthiness to the setting - it either exists or it doesn't. 

If we were adding a new appeal type ABC, we would add `abc` to the wip_docs setting, then use `if wip_doc_enabled?(:abc)` around the places we want to hide on production. So long as that setting exists, it should be hidden from documentation generation unless we also add it to the WIP_DOCS_ENABLED env var.